src/Toast/EventEmitter.js

```diff
@@ -0,0 +1,22 @@
+class EventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  subscribe(event, callback) {
+    if (!this.events[event]) {
+      this.events[event] = [];
+    }
+    this.events[event].push(callback);
+  }
+
+  emit(event, data) {
```

Comment:
[proposal] In the current implementation, when using several Toasts on a page, we observe their simultaneous calling. After clicking on any of the Toast triggers, we simultaneously call the emitter for all toasts on the page. We need to unify this process and make sure that the trigger executes the `showToast` event only for the desired Toast.

On the component usage example pages we have Toast in the code example (copy button) and in the usage example itself.